docs/reference-guides/theme-json-reference/theme-json-living.md

```diff
@@ -83,6 +83,8 @@ Settings related to spacing.
 | margin | boolean | false |  |
 | padding | boolean | false |  |
 | units | array | px,em,rem,vh,vw,% |  |
+| customSpacingSize | boolean | true |  |
+| spacingSizes | array |  | name, size, slug |
 
```

Comment:
Just noticed that the linter's complaining that this needs to be updated with the `spacingScale` property, too 🙂

Reply:
Should be fixed now.